bundle/bundle.go

```diff
@@ -163,12 +142,72 @@ func Read(r io.Reader) (Bundle, error) {
 			if err := util.NewJSONDecoder(&buf).Decode(&bundle.Manifest); err != nil {
 				return bundle, errors.Wrapf(err, "bundle load failed on manifest")
 			}
+
+			if r.includeManifestInData {
+				var metadata map[string]interface{}
+				b, err := json.Marshal(&bundle.Manifest)
+				if err != nil {
+					return bundle, errors.Wrapf(err, "bundle load failed")
+				}
+
+				err = util.UnmarshalJSON(b, &metadata)
+				if err != nil {
+					return bundle, errors.Wrapf(err, "bundle load failed")
```

Comment:
I think it would make sense to add a better exception string here. I can see folks getting quite confused here.
Lets say the tar bundle is correct, but someone messed up the JSON format... it would save some time debugging by saying that the manifest wasn't valid JSON

Reply:
Updated the error messages.